gocd/bci.gocd.yaml.erb

```diff
@@ -148,5 +148,5 @@ pipelines:
         tasks:
         - script: |
             export PYTHONPATH=scripts
-            ./scripts/gocd/bci_repo_publish.py -A https://api.suse.de --verbose --debug run "--token=$BCI_TOKEN" 15-<%= sp %>
+            ./scripts/gocd/bci_repo_publish.py -A https://api.suse.de --verbose --debug run "--repo-token=$BCI_TOKEN" "--rebuild-token=$DEVEL_BCI_REBUILD_TOKEN" 15-<%= sp %>
```

Comment:
You need to feed that environment variable from the secrets store as well. See a couple of lines above

Reply:
ah, missed that, thank you!